docs/reference/aggregations/reducer/movavg-aggregation.asciidoc

```diff
@@ -272,3 +367,10 @@ global trend is slightly positive, so the prediction makes a sharp u-turn and be
 [[double_prediction_global]]
 .Double Exponential moving average with window of size 100, predict = 20, alpha = 0.5, beta = 0.1
 image::images/reducers_movavg/double_prediction_global.png[]
+
+The `holt_winters` model has the potential to deliver the best predictions, since it also incorporates seasonal
+fluctuations into the model:
+
+[[holt_winters_prediction_global]]
+.Holt-Winters moving average with window of size 120, predict = 25, alpha = 0.8, beta = 0.2, gamma = 0.7, period = 30
+image::images/reducers_movavg/triple_prediction.png[]
```

Comment:
It would also be good if throughout all the graphs the data and moving average results stayed the same colour as it gets confusing when they switch (I know this might be a pain to achieve).